### PR TITLE
service: support for the new combo HTTP server

### PR DESCRIPTION
**Pull request by jangko**

### CODE REVIEW

File: templates/nimbus-eth1.service.j2 [hunk 1]
```diff
@@ -28,30 +28,23 @@ ExecStart={{ nimbus_eth1_repo_path }}/build/nimbus \
     --jwt-secret={{ nimbus_eth1_jwt_secret_path | mandatory }} \
 {% endif %}
     --rpc={{ nimbus_eth1_rpc_enabled | to_json }} \
+    --ws={{ nimbus_eth1_ws_enabled | to_json }} \
+    --graphql={{ nimbus_eth1_graphql_enabled | to_json }} \
+{% if nimbus_eth1_rpc_enabled or nimbus_eth1_ws_enabled or nimbus_eth1_graphql_enabled %}
+    --http-address={{ nimbus_eth1_rpc_address }} \
```
Comment (yakimant) [inline templates/nimbus-eth1.service.j2:34]: I see `nimbus_eth1_rpc_address` removed above

Comment (yakimant) [reply templates/nimbus-eth1.service.j2:34]: Same with `nimbus_eth1_rpc_port`

File: templates/nimbus-eth1.service.j2 [hunk 2]
```diff
@@ -28,30 +28,23 @@ ExecStart={{ nimbus_eth1_repo_path }}/build/nimbus \
...
 {% if nimbus_eth1_rpc_enabled %}
     --rpc-api={{ nimbus_eth1_rpc_api | join(",") }} \
-    --rpc-address={{ nimbus_eth1_rpc_address }} \
-    --rpc-port={{ nimbus_eth1_rpc_port }} \
 {% endif %}
-    --ws={{ nimbus_eth1_ws_enabled | to_json }} \
 {% if nimbus_eth1_ws_enabled %}
-    --ws-address={{ nimbus_eth1_ws_address }} \
-    --ws-port={{ nimbus_eth1_ws_port }} \
+    --rpc-api={{ nimbus_eth1_ws_api | join(",") }} \
```
Comment (jakubgs) [inline templates/nimbus-eth1.service.j2:41]:
```suggestion
    --rpc-api={{ nimbus_eth1_rpc_api | join(",") }} \
```

**Review by jakubgs — APPROVED** [review]

**Review by yakimant — APPROVED** [review]

**Comment by yakimant** [comment]
https://github.com/status-im/infra-nimbus/commit/5f4bf2b90f0d3f132fceda1744ab791740241215

**Pull request merged**